perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -92,14 +114,21 @@ def Run(benchmark_spec):
   vms = benchmark_spec.vms
   vm = vms[0]
   logging.info('FIO running on %s', vm)
-  fio_command = '%s --output-format=json --directory=%s %s' % (
-      fio.FIO_PATH, vm.GetScratchDir(), flags.FLAGS.fio_jobfile)
+  if FLAGS.against_device:
+    file_path = vm_util.PrependTempDir(FLAGS.fio_jobfile)
+    fio_command = '--filename=%s %s' % (vm.scratch_disks[0].GetDevicePath(),
```

Comment:
just to double check: the --filename option specified here will apply to all jobs in the job file, right?